DigiMeSDK/Core/Classes/Security/DMEJWTUtility.swift

```diff
@@ -0,0 +1,422 @@
+//
+//  DMEJWTUtility.swift
+//  DigiMeSDK
+//
+//  Created on 04/12/2019.
+//  Copyright © 2019 digi.me Limited. All rights reserved.
+//
+
+import CryptorRSA
+import Foundation
+import SwiftJWT
+
+@objcMembers
+public class DMEJWTUtility: NSObject {
+   
+    // Default JWT header
+    class var header: Header {
+        Header(typ: "JWT",
```

Comment:
We could make use of `Header`'s default initialiser values here and just call `Header(typ: "JWT")`